src/workerd/api/actor.c++

```diff
@@ -109,12 +110,50 @@ jsg::Ref<Fetcher> ColoLocalActorNamespace::get(kj::String actorId) {
       channel, kj::mv(actorId));
   auto outgoingFactory = context.addObject(kj::mv(factory));
 
-  bool isInHouse = true;
-  return jsg::alloc<Fetcher>(
-      kj::mv(outgoingFactory), Fetcher::RequiresHostAndProtocol::YES, isInHouse);
+  return jsg::alloc<DurableObjectRpcStub>(
+      kj::mv(outgoingFactory), Fetcher::RequiresHostAndProtocol::YES, IsInHouse::YES);
 }
 
 // =======================================================================================
+jsg::Promise<jsg::V8Ref<v8::Value>> DurableObjectRpcStub::getRpcAndCallRemote(
```

Comment:
Can this make use of the new `jsg::JsRef<jsg::JsValue>` instead of `jsg::V8Ref<v8::Value>` ... I'm in the process of converting all direct uses of the `v8:` namespace over.